crypto/armcap.c

```diff
@@ -53,6 +53,7 @@ void _armv8_sha1_probe(void);
 void _armv8_sha256_probe(void);
 void _armv8_pmull_probe(void);
 # ifdef __aarch64__
+void _armv8_sm4_probe(void);
```

Comment:
This is a typo, it should be an `_armv8_sm3_probe`

Reply:
Thanks for spotting this issue!  I'll correct it.